.github/workflows/build.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # renovate: tag=v1.0.7
         with:
           profile: minimal
-          toolchain: "1.63.0"
+          toolchain: "1.65.0"
```

Comment:
are there any changes in this PR that require rust 1.65? In the operator-templating changes that we did as preparation for the next release we stayed with [1.63](https://github.com/stackabletech/operator-templating/blob/main/repositories.yaml#L71-L74), so it would be good to know if should move to 1.65 (and we should also change it in ubi8-rust-builder).

Reply:
This should come via templating and not be changed manually (it will fail to build if another templating PR is merged).
@adwk67 could we go for rust 1.65.0 or 1.66.1 in templating before the release?

Reply:
yes, I suggest we go for 1.65. We can make that change before we roll out the templating changes across all operators on Thursday.

Reply:
operator-templating is already on 1.65 https://github.com/stackabletech/operator-templating/commit/8d04aa298954720e38093bace28d9d767c00d2d2
This actually reduces the lag behind upstreaming templating.
IMHO go for it and try to get all the repos up-to-date. Sadly all the templating PRs were closed :/